Gopkg.toml

```diff
@@ -26,7 +26,7 @@ required = [
 
 [[constraint]]
   name = "github.com/percona/pmm"
-  branch = "PMM-2.0"
+  branch = "PMM-4192"
```

Comment:
^

Reply:
I'm not sure that merging pmm repo before other repos is a good idea, it might generate issues for other developers and merge conflicts.

Reply:
In general – yes, but is it true in that particular case?

Besides, merging that PR in pmm repo in Ready for Merge status means there will be changes in that PR _after_ Ready for Merge status.

Up to you and @alexanderblinkov-percona

Reply:
p.s. @BupycHuk There is no point in restarting tests for Percona Server for MySQL 8.0 – they should be updated because PSM was updated :)

Reply:
Yeah, just checked that :)